src/OrleansPSUtils/project.json

```diff
@@ -0,0 +1,18 @@
+{
+  "dependencies": {
+    "Microsoft.CodeAnalysis.Analyzers": "1.0.0",
+    "Newtonsoft.Json": "7.0.1",
+    "Microsoft.CodeAnalysis.Common": "1.0.0",
+    "Microsoft.CodeAnalysis.CSharp": "1.0.0",
+    "System.Collections.Immutable": "1.1.36",
+    "System.Management.Automation.dll": "10.0.10586",
+    "System.Reflection.Metadata": "1.0.21"
+  },
```

Comment:
If this is for using runtime codegen, since we are using transitive restore, I think you might not need to reference it here, you just reference the codegen project, and it should get the dependencies automatically.